agent/roles/manifests/templates/agent-cluster-install_yaml.j2

```diff
@@ -4,7 +4,7 @@ metadata:
   name: test-agent-cluster-install
   namespace: {{ cluster_namespace }}
 spec:
-{% if num_masters != "1" %}
+{% if (platform_type != "none") and (platform_type != "external") %}
```

Comment:
With `external` platform, other scenarios like compact, HA are also possible

Reply:
Yes this fix is necessary to support that, otherwise the VIPs would be set which fails when UserManagedNetwork=true